<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWidget</class>
 <widget class="QWidget" name="ResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBoxRecvResults">
        <property name="title">
         <string>Receiving Results</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutDataAge">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelDataAge">
             <property name="text">
              <string>Last Received:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DataAgeWidget" name="widgetDataAge">
             <property name="font">
              <font>
               <family>DejaVu Sans Mono</family>
              </font>
             </property>
             <property name="text">
              <string>0.0 s</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarReceived">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxGraphControl">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Show</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxForce">
           <property name="text">
            <string>Force</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="radioButtonRaw">
           <property name="text">
            <string>Raw</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QRadioButton" name="radioButtonTranslated">
           <property name="text">
            <string>Translated</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxPressure">
           <property name="text">
            <string>Pressure</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxGridLines">
           <property name="text">
            <string>Grid Lines</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxSave">
        <property name="title">
         <string>Save</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButtonFIRE">
           <property name="text">
            <string>Firing Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDatasheet">
           <property name="text">
            <string>Datasheet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonENG">
           <property name="text">
            <string>RASP ENG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCSV">
           <property name="text">
            <string>CSV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRawCSV">
           <property name="text">
            <string>Raw CSV</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>File Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutFileName">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelFileNameText">
             <property name="text">
              <string>File Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFileName">
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutRecordingDuration">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelRecordingDurationText">
             <property name="text">
              <string>Recording Duration:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelRecordingDuration">
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonBack">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="GraphWidget" name="widgetGraph" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxStats">
       <property name="title">
        <string>Statistics</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutStartupTime">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelStartupTimeText">
            <property name="text">
             <string>Startup Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelStartupTime">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="4">
         <layout class="QHBoxLayout" name="horizontalCStar">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelCStarText">
            <property name="text">
             <string>C*:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCStar">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutMotorDesignation">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelMotorDesignationText">
            <property name="text">
             <string>Motor Designation:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMotorDesignation">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutBurnTime">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelBurnTimeText">
            <property name="text">
             <string>Burn Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBurnTime">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="4">
         <layout class="QHBoxLayout" name="horizontalLayoutAveragePressure">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelAveragePressureText">
            <property name="text">
             <string>Average Pressure:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAveragePressure">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="4">
         <layout class="QHBoxLayout" name="horizontalLayoutPeakPressure">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelPeakPressureText">
            <property name="text">
             <string>Peak Pressure:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPeakPressure">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <layout class="QHBoxLayout" name="horizontalLayoutPeakThrust">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelPeakThrustText">
            <property name="text">
             <string>Peak Thrust:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPeakThrust">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="3">
         <layout class="QHBoxLayout" name="horizontalLayoutAverageThrust">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelAverageThrustText">
            <property name="text">
             <string>Average Thrust:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAverageThrust">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="2">
         <layout class="QHBoxLayout" name="horizontalLayoutISP">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelISPText">
            <property name="text">
             <string>ISP:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelISP">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <layout class="QHBoxLayout" name="horizontalLayoutPropellantMass">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelPropellantMassText">
            <property name="text">
             <string>Propellant Mass:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPropellantMass">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayoutImpulse">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="labelImpulseText">
            <property name="text">
             <string>Total Impulse:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelImpulse">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="3">
         <layout class="QHBoxLayout" name="horizontalLayoutThrustCoefficient">
          <item>
           <widget class="QLabel" name="labelThrustCoefficientText">
            <property name="text">
             <string>Thrust Coefficient:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelThrustCoefficient">
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataAgeWidget</class>
   <extends>QLineEdit</extends>
   <header location="global">ui.dataAgeWidget</header>
  </customwidget>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.graphWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
